client/src/main/java/com/alibaba/nacos/client/identify/STSConfig.java

```diff
@@ -15,7 +15,8 @@
  */
 package com.alibaba.nacos.client.identify;
 
-import org.apache.commons.lang3.StringUtils;
+
```

Comment:
remove extra blank line.

Reply:
In the lasted commit, the extra blank line has aleady been removed.